### PR TITLE
WT-9826 Fix ooo keys error message

### PR DESCRIPTION
**Pull request by etienneptl**
The functions `__bulk_col_keycmp_err` and `__bulk_row_keycmp_err` are called if the compared key is less **or equal** to the previously inserted one. Update the logging message to reflect that.

### CODE REVIEW

File: src/cursor/cur_bulk.c [hunk 1]
```diff
@@ -23,7 +23,7 @@ __bulk_col_keycmp_err(WT_CURSOR_BULK *cbulk)
 
     WT_RET_MSG(session, EINVAL,
       "bulk-load presented with out-of-order keys: %" PRIu64
-      " is less than previously inserted key %" PRIu64,
+      " is less or equal than previously inserted key %" PRIu64,
```
Comment (quchenhao) [inline src/cursor/cur_bulk.c:26]: is less than or equal to

File: src/cursor/cur_bulk.c [hunk 2]
```diff
@@ -200,7 +200,7 @@ __bulk_row_keycmp_err(WT_CURSOR_BULK *cbulk)
     WT_ERR(__wt_scr_alloc(session, 512, &b));
 
     WT_ERR_MSG(session, EINVAL,
-      "bulk-load presented with out-of-order keys: %s compares smaller than previously inserted "
+      "bulk-load presented with out-of-order keys: %s is less or equal than previously inserted "
```
Comment (quchenhao) [inline src/cursor/cur_bulk.c:203]: same

File: src/cursor/cur_bulk.c (hunk 1 above)
Comment (etienneptl) [reply src/cursor/cur_bulk.c]: Thanks see 1435fbed87a92092bf726c0d5a5b0948bcd93f4a

File: src/cursor/cur_bulk.c (hunk 2 above)
Comment (etienneptl) [reply src/cursor/cur_bulk.c]: Thanks see 1435fbed87a92092bf726c0d5a5b0948bcd93f4a

File: src/cursor/cur_bulk.c [hunk 3]
```diff
@@ -23,7 +23,7 @@ __bulk_col_keycmp_err(WT_CURSOR_BULK *cbulk)
 
     WT_RET_MSG(session, EINVAL,
       "bulk-load presented with out-of-order keys: %" PRIu64
-      " is less than previously inserted key %" PRIu64,
+      " is less than or equal than previously inserted key %" PRIu64,
```
Comment (quchenhao) [inline src/cursor/cur_bulk.c:26]: equal to not equal than?

Comment (quchenhao) [reply src/cursor/cur_bulk.c:26]: the previously inserted key, I think

**Review by quchenhao — APPROVED** [review]
lgtm

File: test/suite/test_bulk01.py [hunk 4]
```diff
@@ -162,7 +162,7 @@ def test_bulk_load_order_check(self):
         for i in [1, 9, 10]:
             cursor.set_key(simple_key(cursor, 1))
             cursor.set_value(simple_value(cursor, 1))
-            msg = '/than previously inserted key/'
+            msg = '/less than or equal to previously inserted key/'
```
Comment (quchenhao) [inline test/suite/test_bulk01.py:165]: the? Not a big deal just to be consistent

**Pull request merged**